architecture/maintainer-list.md

```diff
@@ -0,0 +1,10 @@
+## Maintainer List
+
+|                GitHub ID                |  Name  | Company |
+| :-------------------------------------: | :----: | :-----: |
+|   [shenli](https://github.com/shenli)   |  申砾  | PingCAP |
+|  [jackysp](https://github.com/jackysp)  | 于帅鹏 | PingCAP |
+|  [coocood](https://github.com/coocood)  | 周昱行 | PingCAP |
+|  [winkyao](https://github.com/winkyao)  |  姚维  | PingCAP |
+| [zz-jason](https://github.com/zz-jason) | 孙晓光 | PingCAP |
```

Comment:
```suggestion
| [zz-jason](https://github.com/zz-jason) | 孙晓光 | Zhihu |
```

Reply:
the last maintainer should be zhangjian, I fixed the wrong name